subsys/bluetooth/shell/bt.c

```diff
@@ -39,6 +39,9 @@
 #define DEVICE_NAME		CONFIG_BT_DEVICE_NAME
 #define DEVICE_NAME_LEN		(sizeof(DEVICE_NAME) - 1)
 
+/* Multiply bt 1.25 to get MS */
+#define BT_INTERVAL_TO_MS(interval) ((interval) * 5 / 4)
+
```

Comment:
As discussed in another context, this will only give rounded values for the interval.
Still not completely happy about that ...

Is it possible to ask whoever did this change (convert to integer) all-over what the reason is?

(Not that important, though, it is only in the shell.)

Reply:
@jhedberg You made the initial change to this it seems in https://github.com/zephyrproject-rtos/zephyr/commit/cbb7ef55600220a99bc7cac69d66c5acf1fd05f3. Any comments?

Reply:
@Thalley doesnt the commit message explain the reasoning?

Reply:
I think so. @asbjornsabo do you agree?

Reply:
Alternatively, instead of trying to do the conversion in code, the output could be changed to something like:
```
shell_print(ctx_shell, "Interval: 0x%04x x 1.25ms", info.le.interval);
```
I.e. let the user do the calculation if they want to.